android/app/src/main/java/org/fossasia/openevent/common/api/OpenEventAPI.java

```diff
@@ -77,4 +77,7 @@
     @GET("feedbacks?include=user&size=0")
     Observable<List<Feedback>> getFeedbacks();
 
+    @POST("../../feedbacks?include=user,event")
```

Comment:
why `../../`?

Reply:
its that way in the api `../../` to come up the hierarchy `events\{event identifier}` 
<img width="452" alt="screen shot 2018-05-17 at 11 23 33 am" src="https://user-images.githubusercontent.com/16715499/40158961-cd3372e8-59c4-11e8-8387-5a091299a5f3.png">

Reply:
Alright